WordPress/src/main/java/org/wordpress/android/ui/people/RoleChangeDialogFragment.java

```diff
@@ -23,6 +23,15 @@
     private RoleListAdapter mRoleListAdapter;
     private OnChangeListener mListener;
 
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        String role = mRoleListAdapter.getSelectedRole();
+        if (role != null) {
```

Comment:
You're right about the `line 136` it makes it much better to do it the way you've suggested. However for this line, `role` could actually be empty if it wasn't passed during initialization and the user hasn't selected anything yet. So, I think it's good to make sure this is not `null` before writing into the bundle.